pkgs/development/tools/react-native-debugger/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv, fetchurl, unzip, cairo, xorg, gdk_pixbuf, fontconfig, pango, gnome2, atk, gtk2, glib
+, freetype, dbus, nss, nspr, alsaLib, cups, expat, libudev
+}:
+
+let
+  rpath = stdenv.lib.makeLibraryPath [
+    cairo
+    stdenv.cc.cc
+    gdk_pixbuf
+    fontconfig
+    pango
+    atk
+    gtk2
+    glib
+    freetype
+    dbus
+    nss
+    nspr
+    alsaLib
+    cups
+    expat
+    libudev
+
+    gnome2.GConf
+
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXtst
+    xorg.libxcb
+    xorg.libXext
+    xorg.libXi
+    xorg.libXdamage
+    xorg.libXrandr
+    xorg.libXcomposite
+    xorg.libXfixes
+    xorg.libXrender
+    xorg.libXScrnSaver
+  ];
+in stdenv.mkDerivation rec {
+  name = "react-native-debugger-${version}";
+  version = "0.7.18";
+
+  src = fetchurl {
+    url = "https://github.com/jhen0409/react-native-debugger/releases/download/v${version}/rn-debugger-linux-x64.zip";
+    sha256 = "186n438sy9wzrx2zdw4qq4hsz89wiy01bpfa6fdjisvxgz6r8sgw";
+  };
+
+  buildInputs = [ unzip ];
+
+  buildCommand = ''
+    shopt -s extglob
+    mkdir -p $out
+    unzip $src -d $out
+
+    mkdir $out/{lib,bin,share}
+    mv $out/lib{node,ffmpeg}.so $out/lib
```

Comment:
Looks like the tarball bundles libnode.so and libffmpeg.so as binaries. Shoulnd't we use nixpkgs versions instead?

Reply:
Not sure though. I need a second, closer look at the prebuilt package as it could be possible that they're using some kind of modified shared objects

Reply:
No, unfortunately not. This is a package I currently need for some projects and I wanted to file a patch with the expression I originally added to my personal overlay.
The thing is, building from source would require way more effort ATM as we would some kind of builder for electron apps first and regarding such a feature there are currently pending discussions in #26698, #17073 and this project: https://github.com/NixOS/nixpkgs/projects/6
As I don't think I'm sufficiently qualified for this job (and I don't think that I'll have enough time in the next months) I didn't work on this, however I'd rewrite the expression in case we have such a feature.